src/GraphQL.ApiTests/net50/GraphQL.approved.txt

```diff
@@ -660,6 +660,11 @@ namespace GraphQL
     {
         public OutputTypeAttribute() { }
     }
+    public static class QueryArgumentExtensions
+    {
+        public static GraphQL.Types.QueryArgument ParseValue(this GraphQL.Types.QueryArgument argument, System.Func<object, object> parseValue) { }
+        public static GraphQL.Types.QueryArgument Validate(this GraphQL.Types.QueryArgument argument, System.Action<object> validator) { }
+    }
```

Comment:
These are extension methods directly on `QueryArgument` because we have no builder classes for arguments.  Rather, the configure delegate of `.Argument<int>("age", configure: arg => { })` passes the `QueryArgument` directly.